.github/workflows/wheels.yml

```diff
@@ -31,18 +31,79 @@ jobs:
           path: ./dist
 
 
-  make_wheels:
-    name: ${{ matrix.name }}
+  make_cibw_v1_wheels:
+    name: "cibuildwheel v1: ${{ matrix.name }}"
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
+      matrix:
+        # Only build for platforms that cibw v2 doesn't support
+        include:
+        - os: macos-latest
+          build: "cp27-macosx_x86_64"
+          name: macOS Intel
+        - os: windows-latest
+          build: "cp27-win_amd64"
+          name: Windows 64-bit
+        - os: ubuntu-latest
+          build: "cp27-manylinux_x86_64"
+          name: Linux Intel 64-bit
+
+    steps:
+
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      ## Setup Env
+
+      - uses: ilammy/msvc-dev-cmd@v1.10.0
+        if: runner.os == 'Windows'
+
+      - name: Set Windows variables
+        if: runner.os == 'Windows'
+        shell: bash
+        run: |
+          echo "CC=cl.exe" >> $GITHUB_ENV
+          echo "CXX=cl.exe" >> $GITHUB_ENV
+
+      - name: Set up QEMU
+        if: runner.os == 'Linux'
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: aarch64
```

Comment:
I'm guessing this is only relevant for aarch64 builds, and so can be removed for cibw v1 wheels.